sdk/storage/tests.yml

```diff
@@ -27,5 +27,5 @@ jobs:
         STORAGE_DATA_LAKE_ACCOUNT_KEY: $(STORAGE_DATA_LAKE_ACCOUNT_KEY)
         AZURE_STORAGE_FILE_ACCOUNT_NAME: $(AZURE_STORAGE_FILE_ACCOUNT_NAME)
         AZURE_STORAGE_FILE_ACCOUNT_KEY: $(AZURE_STORAGE_FILE_ACCOUNT_KEY)
-        AZURE_STORAGE_QUEUE_ACCOUNT_NAME: $(AZURE_STORAGE_FILE_ACCOUNT_NAME)
+        AZURE_STORAGE_QUEUE_ACCOUNT_NAME: $(AZURE_STORAGE_QUEUE_ACCOUNT_NAME)
```

Comment:
From our latest updates to the resource provisioning scripts you do not need to do the variable indirection here. All of the `EnvVars` of the form `AZURE_STORAGE_QUEUE_ACCOUNT_NAME: $(AZURE_STORAGE_QUEUE_ACCOUNT_NAME)` can be removed. 

You'll only need to keep the variables that are not set by the ARM template (`AZURE_TEST_MODE`, `AZURE_TENANT_ID`, `AZURE_CLIENT_ID`, `AZURE_CLIENT_SECRET`)